app/models/spree/avalara_transaction.rb

```diff
@@ -166,6 +167,11 @@ def tax_calculation_enabled?
       Spree::Config.avatax_tax_calculation
     end
 
+    def resolve_location_code
+      default_stock_location = Spree::StockLocation.find_by name: 'Mejuri'
+      order.purchase_location.stock_location.avalara_location_code || default_stock_location.avalara_location_code
```

Comment:
Nitpick: only look up the default stock location if `avalara_location_code` cannot be found. Maybe extracting `default_avalara_location` to a method?